processor/eqReceiptProcessor.go

```diff
@@ -0,0 +1,52 @@
+package processor
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/ONSdigital/census-rm-pubsub-adapter/config"
+	"github.com/ONSdigital/census-rm-pubsub-adapter/models"
+	"github.com/pkg/errors"
+)
+
+type EqReceiptProcessor struct {
+	*Processor
+}
+
+func NewEqReceiptProcessor(ctx context.Context, appConfig *config.Configuration) *EqReceiptProcessor {
```

Comment:
I suppose I should have questioned this in the earlier PR, and the fact it's tested and stuff makes me question myself, but isn't passing a _copy_ of the `ctx` struct going to undermine its usefulness as a cross-goroutine mechanism to communicate, given that each one of these processors is going to have its own copy of the `Context` instead of all pointing at the same shared one?

Reply:
Same situation as the message, `context.Context` is an interface.